bukuserver/templates/bukuserver/lib.html

```diff
@@ -17,8 +17,9 @@
 {% macro limit_navigation_if_popup() %}
   <style>
     .popup .navbar-brand {pointer-events: none}
-    .popup :is(#admin-navbar-collapse, .navbar-toggle) {display: none !important}
-    .popup .nav-tabs :is(:first-child, :nth-child(2):not(.active)) > * {display: none}
+    .popup #admin-navbar-collapse {display: block}
+    .popup :is(.navbar-nav, .navbar-toggler) {display: none}
+    .popup .nav-tabs :is(:first-child, :nth-child(2)) > .nav-link:not(.active) {display: none}
```

Comment:
`#admin-navbar-collapse` is now the style of the entire header (it simply changes `display:` with page width); and the overall design of navbar/tabs got changed as well